frontend/webapp/reuseable-components/status/active-status/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Status, type StatusProps } from '@/reuseable-components';
+
+interface Props extends StatusProps {}
+
+export const ActiveStatus: React.FC<Props> = ({ isActive, ...props }) => {
+  return <Status title={isActive ? 'Active' : 'Inactive'} isActive={isActive} {...props} />;
```

Comment:
do we need two more files for active and connection status if they only hold the status component?

Reply:
`Status` is a base component, it has a variety of style options, ranging from colors, fonts, borders, and backgrounds.

`ActiveStatus`, `ConnectionStatus`, and `InstrumentStatus` all leverage `Status`, but apply different styles.

In the case of `ActiveStatus`, we have 1 version with icon + border, and 1 version with nothing.
In the case of `ConnectionStatus`, we have 1 version with icon + background.
In the case of `InstrumentStatus `, we have 1 version with icon + border, and "pale" if inactive.

So yes, we need these to be separated.